package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "api.goodchat",
+  "version": "1.0.0",
+  "description": "omnichannel chat solution built for structured integrations with transactional systems",
+  "main": "index.js",
+  "scripts": {
+    "build": "tsc",
+    "test": "TS_NODE_PROJECT=tsconfig.spec.json nyc mocha -r ts-node/register 'specs/**/*.spec.ts'",
+    "coverage:json": "nyc report --reporter=json",
+    "coverage:html": "nyc report --reporter=html"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/patrixr/api.goodchat.git"
```

Comment:
you can update these now you've moved repo over to crossroads

Reply:
good catch :)